doc/user/content/ingest-data/sql-server/self-hosted.md

```diff
@@ -113,8 +113,7 @@ scenarios, we recommend separating your workloads into multiple clusters for
 {{< note >}}
 For a new SQL Server source, if none of the replicating tables
 are receiving write queries, snapshotting may take up to an additional 5 minutes
-to complete. For details, see [Snapshot latency for inactive
-databases](#considerations).
+to complete. For details, see [snapshot latency for inactive databases](/sql/create-source/sql-server/#snapshot-latency-for-inactive-databases).
```

Comment:
If we could modify 
https://github.com/MaterializeInc/materialize/blob/self-managed-docs/v25.2/doc/user/shared-content/sql-server-considerations.md
to include the snapshot latency info .. then, we can do the same here and link to section on the page.